cross-modal-search/flows/flow-index.yml

```diff
@@ -34,9 +34,25 @@ pods:
     needs: [gateway]
   - name: text_indexer
     polling: any
-    uses: pods/index-text.yml
+    #uses: pods/index-text.yml (numpy + binary pb indexer)
+    uses: NumpyIndexer
     shards: $JINA_SHARDS
+  - !NumpyIndexer
```

Comment:
Can you add comments on the yml file of what is going on? As in the [other examples](https://github.com/jina-ai/examples/issues/445)

Reply:
Sure, done